granulate_utils/linux/cgroups/memory_cgroup.py

```diff
@@ -18,14 +18,23 @@ def get_memory_limit(self) -> int:
     def get_max_usage_in_bytes(self) -> int:
         return int(self.read_from_control_file(self.max_usage_in_bytes))
 
-    def set_limit_in_bytes(self, limit: int) -> None:
-        self.write_to_control_file(self.limit_in_bytes, str(limit))
+    def _set_memsw_limit_in_bytes(self, limit: int) -> None:
         try:
             self.write_to_control_file(self.memsw_limit_in_bytes, str(limit))
         except PermissionError:
```

Comment:
Any other exception should be raised here?

Reply:
PermissionError means the file doesn't exist and this is the case we are handling here - imo if it's another exception we need to raise on it in order not to lose it. @d3dave wdyt?

Reply:
+ I place a comment just under it explaining the reason